moler/util/moler_test.py

```diff
@@ -170,12 +171,11 @@ def _prepare_err_msg(cls, caught_exception):
         if len(occured_exceptions) > 0:
             err_msg += "There were unhandled exceptions from test caught by Moler.\n"
             for i, exc in enumerate(occured_exceptions, 1):
-                try:
-                    import traceback
+                if hasattr(exc, '__traceback__'):
                     exc_traceback = ' '.join(traceback.format_tb(exc.__traceback__))
-                    err_msg += "  {}) {}{}\n".format(i, exc_traceback, repr(exc))
-                except AttributeError:
-                    err_msg += repr(exc)
+                    err_msg += "  ({}) {}{}\n".format(i, exc_traceback, repr(exc))
+                else:
+                    err_msg += "  {}\n".format(cls._get_traceback())
```

Comment:
In a loop it will add to logs all tracebacks. Maybe set the flag and call the method _get_traceback outside the loop?